src/app/index.scss

```diff
@@ -1,6 +1,7 @@
 // FIXME should use normalize.scss version instead
 // but I got some errors so I don't want to spend too much time here now.
 @import "../assets/normalize.scss";
```

Comment:
Not related to @vlandham's work, but it seems like we *do* use `normalize.scss` here—is this comment out of date?